setup.py

```diff
@@ -1,7 +1,15 @@
 try:
-    from setuptools import setup #Py2
+    from setuptools import setup  #Py2
 except ImportError:
-    from distutils.core import setup #Py3
+    from distutils.core import setup  #Py3
+
+import sys
+
+install_requires = []
+if sys.version_info < (3, 4):
+    install_requires.append('enum34')
+
+print(install_requires)
```

Comment:
left over?

Reply:
Yes, my bad. pushed a fix.

Reply:
Can you please squash the two commits? Thanks!

Reply:
Forced a push.